db/migrations/00002_add_edges_table.sql

```diff
@@ -0,0 +1,48 @@
+-- +goose Up
+-- +goose StatementBegin
+
+-- Create edges table
+CREATE TABLE edges (
+    from_resource_type_int INTEGER NOT NULL,       -- This is the proto ID
```

Comment:
nit:
from_resource_proto_type? I'm fine with it as is with the comment as well, but this might make it hard to confuse that it's proto related